### ISSUE TITLE
Maester - Teams Tests - Fixes

### ISSUE THREAD

**Issue by merill** [maester365/maester#569]
This issue is to track the updates planned for the Maester - Teams tests cc @weyCC81 

1. Move each test into Maester cmdlets
2. Provider a reference number for each test e.g. MT1xxx (like we do with the other Maester tests)
3. Include the reference number as a tag
4. Improve the markdown to give guidance on the steps to get to the blade to make the change (including the descriptive label) and the Recommended Action step.

**Comment by merill** [comment]
@weyCC81 I've added references and updated the first Teams test as a template.

* Added Test-MtTeamsRestrictParticipantGiveRequestControl
* Added .md file
* Instructions on how to fix

Would you be able to update the rest of the tests to follow the format.

PS: A future enhancement would be to add the Teams related CIS tests. Maybe @NZLostboy has thoughts. I think some of these are very similar.

**Comment by NZLostboy** [comment]
@merill Yeah absolutely, sorry crazy end to the year and I haven't had time to finish the CIS checks.

I have some time put aside after next week to power through and finish all the CIS checks. Where we can combine the Teams tests into one test that makes total sense, but I am trying to keep the CIS tests representative of the documentation where possible so its a balance. Caching them where we can would be the bare minimum, so we can definitely achieve that. 

I can pick up the state end next week when things settle down and get stuck in. Happy to collaborate here or Discord to work together on this

**Comment by merill** [comment]
Agree, we do need to keep the rules isolated so the CIS ones can track against the CIS definitions. 

I would want to avoid adding the same test to the Maester core tests if they are a simple dup.
